File: tests/perf/simple_test.py
"""
Kyumins-MacBook-Pro:xc math4tots$ time python simple_test.py 
100000020000000

real	0m3.530s
user	0m3.202s
sys	0m0.303s
"""

xs = []
for i in range(10000000):
  xs.append(2 * i + 3)

total = 0
for x in xs:
  total += x

print(total)

File: tests/perf/simple_test.java
/*
Kyumins-MacBook-Pro:xc math4tots$ javac simple_test.java && time java Test
100000020000000

real	0m2.719s
user	0m12.256s
sys	0m0.390s
*/
import java.util.ArrayList;

public class simple_test {
  public static void main(String[] args) {
    ArrayList<Long> xs = new ArrayList<Long>();
    for (long i = 0; i < 10000000; i++) {
      xs.add(2 * i + 3);
    }

    long total = 0;
    for (long i: xs) {
      total += i;
    }

    System.out.println(total);
  }
}
